caddyconfig/httpcaddyfile/httptype.go

```diff
@@ -820,6 +820,9 @@ func (st *ServerType) serversFromPairings(
 			}
 		}
 
+		// sort for deterministic JSON output
+		slices.Sort(srv.Logs.SkipHosts)
```

Comment:
On mobile... Should it be this?

```suggestion
if srv.Logs != nil {
		slices.Sort(srv.Logs.SkipHosts)
}
```

Reply:
That's probably safer, yeah

Reply:
Yeah I made this then walked away without looking at CI, will fix in a bit